superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -66,8 +75,8 @@ class ChartContainer extends React.Component {
       || nextProps.height !== this.props.height);
   }
 
-  componentDidUpdate() {
-    this.renderVis();
+  componentDidUpdate(prevProps) {
```

Comment:
I think renderVis uses a ref to ChartContainer, so it has to happen after mount. And setState should be put into willReceiveProps.